cmd/varint/flag.go

```diff
@@ -0,0 +1,31 @@
+package main
```

Comment:
My suggestion: Let's round out our CLI tools suite and then see what makes best sense to factor out into some sort of CLI-tool library.